practice-one/src/assets/styles/layouts/header.css

```diff
@@ -0,0 +1,164 @@
+.header__section {
+  position: absolute;
+  background-image: url(/practice-one/src/assets/images/Background-header.png);
+  height: 768px;
+  flex-shrink: 0;
+}
+
+.header__section--navbar {
+  display: flex;
+  width: 1140;
+  height: 45px;
+  justify-content: space-between;
+  margin-top: 25px;
+  margin-right: 152px;
+  margin-left: 152px;
+}
+
+.text-logo {
+  color: var(--color-primary);
+  text-align: center;
+  font-size: 36px;
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-default);
+  letter-spacing: 1.997px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin-top: 5px;
+  gap: 70px;
+}
+
+.text-list {
+  color: var(--color-base);
+  text-decoration: none;
+  align-items: center;
+  font-size: 18px;
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-default);
+  letter-spacing: 1px;
+}
+
+.list-icon {
+  display: flex;
+  align-items: center;
+  gap: 25px;
+}
+
+.icon-search {
+  width: 24.02px;
+  height: 31.808px;
+  flex-shrink: 0;
+}
+
+.text-overlay {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: rgb(254, 247, 247);
+}
+
+.icon-container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.banner-section {
+  position: absolute;
+  display: grid;
+  grid-template-columns: 1.5fr 1fr;
+  grid-template-rows: repeat(3, minmax(auto, auto));
+  width: 100%;
+  bottom: 0;
+}
+
+.banner__section--decris {
+  display: flex;
+  height: 600px;
+  flex-direction: column;
+}
+
+.iphone {
+  display: flex;
+  bottom: 0;
+}
+
+.text-ui {
```

Comment:
Is that common class? this class name is not a good name. Please find other name